makefiles/shared.mk

```diff
@@ -77,9 +77,14 @@ next-gen-stage: ## Host online for review
 	fi
 endif
 
+ifeq ($(MUT_PREFIX),)
+SITE_URL=${URL}/${PROJECT}/${USER}/${BRANCH_NAME}
```

Comment:
(Nit; non-blocking) Should we consider naming `BRANCH_NAME` -> `GIT_BRANCH` to help consolidate variable names? Or would doing this be a bad idea?

Reply:
This is a legitimate question. I didn't want to overlap with anything because of the speed of this fix needed and, like MUT_PREFIX, I didn't know if GIT_BRANCH would be available at time of the oas build either.

I can start experimenting with this maybe in the future?

Reply:
Sure, sounds good.